packages/nuxt/src/inspector.ts

```diff
@@ -0,0 +1,24 @@
+import { useLogger, useNuxt } from '@nuxt/kit'
+import { withTrailingSlash, withoutTrailingSlash } from 'ufo'
+import * as chalk from 'chalk'
+import pluginInspector from '../../inspector/node'
+
+export function addInspector() {
+  const nuxt = useNuxt()
+  const inspectorRoute = '/_unocss'
+
+  if (nuxt.options.dev && nuxt.options.builder === '@nuxt/vite-builder') {
+    nuxt.options.vite.plugins = nuxt.options.vite.plugins || []
+    nuxt.options.vite.plugins.push(pluginInspector({} as never))
```

Comment:
Isn't the Vite plugin already enabling the inspector? I am not sure if this is needed here. We might just need to pass the option to Vite plugin

Reply:
what is the option you are referring to?